src/schema/v2/notifications/__tests__/index.test.ts

```diff
@@ -13,6 +13,7 @@ describe("notificationsConnection", () => {
           date: "2022-08-22T21:15:49.000Z",
           object_ids: ["63036fafbe5cfc000cf358e3", "630392514f13a5000b55ecec"],
           object: { artist: { id: "damien-hirst" } },
+          activity_type: "ARTWORK_ALERT",
```

Comment:
**suggestion**: I think this should actually be `SavedSearchHitActivity`. The enum keys are used at the GraphQL layer but from Gravity, we use the underlying string value.

Reply:
aha yes thank you, good catch! updated